src/epd2in13_v2/constants.rs

```diff
@@ -33,3 +35,52 @@ pub(crate) const LUT_PARTIAL_UPDATE: [u8; 70] =[
     0x00,0x00,0x00,0x00,0x00,                       //  TP5 A~D RP5
     0x00,0x00,0x00,0x00,0x00,                       //  TP6 A~D RP6
 ];
+
+#[cfg(feature = "epd2in13_v3")]
+#[rustfmt::skip]
+// Original Waveforms from Waveshare
+pub(crate) const LUT_PARTIAL_UPDATE: [u8; 159] =[
```

Comment:
In waveshare repo (https://github.com/waveshare/e-Paper/blob/60762ac5a3787ca7c3080d0e1f256a32ec3147e6/Arduino/epd2in13_V3/epd2in13_V3.cpp#L176) I see:

```cpp
void Epd::Lut(unsigned char *lut)
{
	unsigned char count;
	SendCommand(0x32);
	for(count = 0; count < 153; count++) {
		SendData(lut[count]);
	}

	SendCommand(0x3f);
	SendData(*(lut+153));
	SendCommand(0x03);	// gate voltage
	SendData(*(lut+154));
	SendCommand(0x04);	// source voltage
	SendData(*(lut+155));	// VSH
	SendData(*(lut+156));	// VSH2
	SendData(*(lut+157));	// VSL
	SendCommand(0x2c);		// VCOM
	SendData(*(lut+158));
}
```

They're setting the first 153 bytes as LUT. Is it intended?